disp_driver/include/display.hpp

```diff
@@ -0,0 +1,29 @@
+#ifndef DISPLAY_HPP_
+#define DISPLAY_HPP_
+
+#include "esp_err.h"
+#include "freertos/FreeRTOS.h"
+#include "freertos/task.h"
+#include "lvgl.h"
+#include "lvgl_helpers.h"
+
+#define BACKLIGHT_GPIO      7

```

Comment:
This is fine for now, but do you think a Kconfig section for GPIO pins would be good to have?